### ISSUE TITLE
[Serious] Focus indicator lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#633]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87016 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - States of User Interface Components (1.4.11.b) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Jobs List |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 04/21/26 at 12:10 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | sharyn.morrow@ablr360.com |

## Description

Focus indicator lacks 3 to 1 contrast ratio.
The custom focus indicator on the blue New button does not meet minimum color contrast requirements. When the button receives keyboard focus, the visible focus style is rendered as a light blue outline against a white background with a contrast ratio of 1.8:1 which is below the required 3:1 minimum.

## Recommendation

Rule : 
The contrast of visual focus indicators against the background MUST be at least 3 to 1.

How To Fix: 
Fix this issue by adjusting the visual focus indicator to achieve 3 to 1 contrast according the scenarios below.

When a focus indicator appears:

1. Outside the component, it needs to contrast with the background that the component is on.
2. Inside the component, it needs to contrast with the adjacent color(s) within the component.
3. As the border of the component (inside the component and adjacent to the outside), it needs to contrast with both adjacent colors.
4. Appears partly inside and partly outside, either part of the focus indicator can contrast with the adjacent colors.

Reference: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

Background: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

## Screenshots

![04_visible-focus-contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87016/04_visible-focus-contrast.png)
